Apps/WebClient/src/ClientApp/app/views/timeline.vue

```diff
@@ -551,6 +539,7 @@ export default class TimelineComponent extends Vue {
     this.isAddingNote = false;
     if (note) {
       this.timelineEntries.push(new NoteTimelineEntry(note));
+      this.sortEntries();
```

Comment:
shouldn't we move this to the applyTimelineFilter method since we are calling both of them in all those cases?